members/I000055.yaml

```diff
@@ -54,11 +54,11 @@ contact_form:
         options:
           max_length: 10000
     - javascript:
-        - name: Message
-          selectors: [ "textarea#field_4BBB4940-5FA9-43E1-9F05-0A9EB169C378" ]
-          values: [ "$MESSAGE" ]
-          commands: [ "elements[0].value = values[0].replace(/www\\.youtube\\.com\\/watch?[^&]*(&)?v=/gi, 'youtu.be/')" ]
-          required: true
+      - name: Message
+        selectors: [ "textarea#field_4BBB4940-5FA9-43E1-9F05-0A9EB169C378" ]
+        values: [ "$MESSAGE" ]
+        commands: [ "elements[0].value = elements[0].value.replace(/www\\.youtube\\.com\\/watch?[^&]*(&)?v=/gi, 'youtu.be/')" ]
```

Comment:
is it `elements[0].value.replace` or `values[0].replace`?

Reply:
per Tommy's edit, is ```
elements[0].value.replace
```

Reply:
So should other instances of `values[0].replace` be removed?

Reply:
No, since this yaml has a previous $MESSAGE variable inside fill_in, the `elements[0].value.replace` in the JS steps ensures that it does not overwrite the first message variable. This is the only place where this would work, as we are leaving the max_length option above inside fill_in for it to handle the appropriate length. For the other yamls, `values[0].replace` would be fine since we are just dealing with one $MESSAGE variable and not max_length and adding the original blacklisted character to the JS step. 

Hope this makes sense, although I'm sure @tirish could explain it more succinctly

Reply:
@OdeclasV disregard my comments from just now - I see the blacklist handled in the JS step